src/components/BookItem.js

```diff
@@ -9,19 +12,56 @@ const BookItem = (prop) => {
     dispatch(delBook(id));
   };
   return (
-    <div>
-      <li>
-        <h3>{title}</h3>
-        <span>
-          {author}
-        </span>
-        <p>{category}</p>
-        <button type="button" onClick={handleDelete}>
-          Delete
+    <li>
+      <div className="book">
+        <div className="d-flex details">
+          <p className="current-chapter">{category}</p>
+          <h3 className="title">{title}</h3>
+          <span>
+            {author}
+          </span>
+          <div>
+            <button type="button" onClick={() => { }} className="btn-single">
+              Comment
+            </button>
+            <button type="button" onClick={handleDelete} className="btn-single">
+              Remove
+            </button>
 
-        </button>
-      </li>
-    </div>
+            <button type="button" onClick={() => { }} className="btn-single">
+              Edit
+            </button>
```

Comment:
- [ ] kindly consider sticking to the given design as much as possible to make your project an exact duplicate design of the given one check on the image below for the highlighted point to fix 😀 
<img width="1438" alt="Screen Shot 2023-06-24 at 21 41 55" src="https://github.com/hiwina/bookstore/assets/60146030/9ecec00f-4883-48d4-a3f6-da67e0709f43">